pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -305,6 +305,78 @@ spec:
 			})
 		})
 
+		t.NewSubTest("verify that namespaces without istio-enable label are not included to the SMMR list").Run(func(t test.TestHelper) {
+			t.LogStep("Create 5 member namespaces")
+			createMemberNamespaces(t, 5)
+
+			t.LogStep("Wait for SMMR to be Ready")
+			oc.WaitSMMRReady(t, meshNamespace)
+			t.Logf("Actual SMMR members: %s", meshNamespace)
+
+			t.LogStep("Remove istio-injection=enabled label from member-1 and member-3 namespaces")
+			for _, member := range []string{"member-0", "member-2", "member-4"} {
+				t.Logf("Removing label from namespace %s", member)
+				oc.RemoveLabel(t, "", "Namespace", member, "istio-injection")
+			}
+
+			t.LogStep("Add other variation of istio-injection value to member-2 and member-4 namespace")
```

Comment:
Maybe simplify the definition of the step saying: `Add again the label istio-injection to the namespaces....`

Reply:
Done